Bottom: 6c5150f426cfc3836d7a99218f91adbaa7669ebf
Top:    2acaddac00697b8ae535ce41d55be1517dd8b115
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-10 15:23:08 +0800

Refresh of vhost-hide-used-ring-layout

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index b4378c9..a8c9506 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -516,7 +516,7 @@ static void handle_tx(struct vhost_net *net)
 		busyloop_intr = false;
 		err = vhost_net_tx_get_vq_desc(net, vq, &used, vq->iov,
 					       ARRAY_SIZE(vq->iov),
-					       &out, &in);
+					       &out, &in, &busyloop_intr);
 		/* Nothing new?  Wait for eventfd to tell us they refilled. */
 		if (err == -ENOSPC) {
 			if (unlikely(busyloop_intr)) {
